libs/auth-api-lib/src/lib/resources/dto/api-scope-user-update.dto.ts

```diff
@@ -3,6 +3,13 @@ import { IsNotEmpty, IsOptional, IsString } from 'class-validator'
 import { ApiScopeUserAccessDTO } from './api-scope-user-access.dto'
 
 export class ApiScopeUserUpdateDTO {
+  @IsOptional()
+  @IsString()
+  @ApiProperty({
+    example: 'Jane Doe',
+  })
+  name!: string
```

Comment:
Shouldn't this be undefined if missing? Same with the email property below.
```suggestion
  name?: string
```